src/vaults/BaseInvestmentManager.sol

```diff
@@ -51,6 +51,7 @@ abstract contract BaseInvestmentManager is Auth, IBaseInvestmentManager {
         IBaseVault vault_ = IBaseVault(vaultAddr);
         VaultDetails memory vaultDetails = poolManager.vaultDetails(address(vault_));
 
+        // TODO(wischli): Check if should be overwritten by SyncRequests using the custom valuation
```

Comment:
Suppose the `SyncRequests` should also overwrite the three inherited views from the `BaseInvestmentManager` which use the `poolManager.priceAssetPerPool`?!

Reply:
Override applied for `convertToShares` and `convertToAssets` in [4aab9c5](https://github.com/centrifuge/protocol-v3/pull/184/commits/4aab9c50da48f553d5ce8b79d54ccd56a70e7558). For `priceLastUpdated` it didn't seem to make sense because the custom valuation doesn't support that.

Do you agree @mustermeiszer?